EIPS/eip-7876.md

```diff
@@ -0,0 +1,636 @@
+---
+eip: 7876
+title: Ethereum Network Configuration for DApps
+description: EVM-compatible network configuration specification for DApps supporting any custom network features through extensions.
+author: Bogdan Gusiev (@bogdan), Sergey Bomko (@aquiladev)
+discussions-to: https://ethereum-magicians.org/t/eip-7876-unified-network-configuration/22763
+status: Draft
+type: Informational
+created: 2025-01-03
+requires: 55
+---
+
+## Abstract
+
+This standard defines a universal format for specifying network configurations to decentralized applications (DApps). The configuration includes essential information about Ethereum networks, such as available RPC URLs, native currencies, contract addresses, and block explorers. Configurations are intended to be provided by:
+
+- Network maintainers to make sure their network is supported by DApp.
+- Smart contract developers to ensure the smart contracts are properly integrated into DApps and changes are consistently delivered.
+- DApps developers internally in the organisation to make use of software libraries designed around the standard and maintain consistent configuration across multiple platforms (e.g iOS, Android).
+
+The goal of this standard is to simplify network configuration for DApp developers and enhance interoperability across Ethereum-compatible networks.
+
+Given that different EVM-compatible chains introduce unique features, this standard also supports extensibility, allowing networks to specify additional configuration parameters beyond the base requirements. This flexibility prevents networks from resorting to external storage for their unique configurations, preserving the goal of a unified and comprehensive network configuration standard.
+
+---
+
+## Motivation
+
+Currently, decentralized applications (DApps) support numerous EVM-compatible networks and often define network configurations in inconsistent formats, which complicates interoperability and sharing of configurations. This standard introduces a **unified network configuration** format that can be adopted by EVM-based networks, making it easier for developers to integrate with multiple networks and providing consistent configuration management.
+
+The proposed configuration format includes essential details required by DApps, including:
+
+- Network name and ID
+- RPC URL for Ethereum nodes
+- Native currency details
+- Smart contract addresses
+- Available block explorers and NFT marketplaces
+
+---
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+The configuration **MUST** use a JSON object, which DApp developers can use to define multiple Ethereum networks.
+
+---
+
+### Interface definition
```

Comment:
I'd like to see a rendered example. It's somewhat hard to imagine what the final structure will look like from just the TypeScript definitions. You could move the one from Reference Implementation up here.